confignode/src/main/java/org/apache/iotdb/confignode/manager/IManager.java

```diff
@@ -264,4 +267,10 @@ TDataPartitionResp getOrCreateDataPartition(
 
   /** Show datanodes */
   DataSet showDataNodes();
+
+  TSStatus createSchemaTemplate(TCreateSchemaTemplateReq req);
+
+  TGetAllTemplatesResp getAllTemplates();
+
+  TGetTemplateResp getTemplate(String req);
```

Comment:
Add interface annotation to show what these interfaces do

Reply:
done